tests/page-objects/login/login.po.js

```diff
@@ -20,6 +21,17 @@ const changeLocale = locale => {
   return element(by.css(`.locale[name="${locale}"]`)).click();
 };
 
+const getLabelForUser = () => {
+  return element(by.css('#form > label:nth-child(2)'));
+};
+const getLabelForPassword = () => {
+  return element(by.css('#form > label:nth-child(4)'));
```

Comment:
Same as above about the 'for' attribute. 

`element(by.css('#form > label[for="password"]'));`